sdk/src/main/scala/com/horizen/consensus/TimeToEpochSlotConverter.scala

```diff
@@ -19,6 +19,11 @@ trait TimeToEpochSlotConverter {
       intToConsensusSlotNumber(slotIndex.toInt + 1)
   }
 
+  def timeStampToAbsoluteSlotNumber(timestamp: Block.Timestamp): ConsensusSlotNumber = {
```

Comment:
Add comment, what is Absolute slot number.
Define ConsensusSlotNumber -> ConsensusAbsoluteSlotNumber, because we can't compare AbsoluteSlot with Slot.

For future discussion: probably have case class (epoch, slot) and comparator inside it. Practically, `timeStampToSlotNumber` and `timeStampToEpochNumber` are bounded: when we parse "timestamp" we at first get epoch, than slot. Note: no sense to do such changes in current PR.

Reply:
done